// wagmi/dist/esm/exports/index.js
////////////////////////////////////////////////////////////////////////////////
// Context
////////////////////////////////////////////////////////////////////////////////
// biome-ignore lint/performance/noBarrelFile: entrypoint module
export { WagmiContext, WagmiProvider, } from '../context.js';
////////////////////////////////////////////////////////////////////////////////
// Errors
////////////////////////////////////////////////////////////////////////////////
export { BaseError } from '../errors/base.js';
export { WagmiProviderNotFoundError, } from '../errors/context.js';
////////////////////////////////////////////////////////////////////////////////
// Hooks
////////////////////////////////////////////////////////////////////////////////
export { useBalance, } from '../hooks/useBalance.js';
export { useBlock, } from '../hooks/useBlock.js';
export { useBlockNumber, } from '../hooks/useBlockNumber.js';
export { useBlockTransactionCount, } from '../hooks/useBlockTransactionCount.js';
export { useBytecode, } from '../hooks/useBytecode.js';
export { useCall, } from '../hooks/useCall.js';
export { useCallsStatus, } from '../hooks/useCallsStatus.js';
export { useCapabilities, } from '../hooks/useCapabilities.js';
export { useChainId, } from '../hooks/useChainId.js';
export { useChains, } from '../hooks/useChains.js';
export { useClient, } from '../hooks/useClient.js';
export { useConfig, } from '../hooks/useConfig.js';
export { useConnect, } from '../hooks/useConnect.js';
export { 
/** @deprecated use `useConnection` instead */
useConnection as useAccount, useConnection, } from '../hooks/useConnection.js';
export { 
/** @deprecated use `useConnectionEffect` instead */
useConnectionEffect as useAccountEffect, useConnectionEffect, } from '../hooks/useConnectionEffect.js';
export { useConnections, } from '../hooks/useConnections.js';
export { useConnectorClient, } from '../hooks/useConnectorClient.js';
export { useConnectors, } from '../hooks/useConnectors.js';
export { useDeployContract, } from '../hooks/useDeployContract.js';
export { useDisconnect, } from '../hooks/useDisconnect.js';
export { useEnsAddress, } from '../hooks/useEnsAddress.js';
export { useEnsAvatar, } from '../hooks/useEnsAvatar.js';
export { useEnsName, } from '../hooks/useEnsName.js';
export { useEnsResolver, } from '../hooks/useEnsResolver.js';
export { useEnsText, } from '../hooks/useEnsText.js';
export { useEstimateFeesPerGas, } from '../hooks/useEstimateFeesPerGas.js';
export { useEstimateGas, } from '../hooks/useEstimateGas.js';
export { useEstimateMaxPriorityFeePerGas, } from '../hooks/useEstimateMaxPriorityFeePerGas.js';
export { useFeeHistory, } from '../hooks/useFeeHistory.js';
export { useGasPrice, } from '../hooks/useGasPrice.js';
export { useInfiniteReadContracts, } from '../hooks/useInfiniteReadContracts.js';
export { usePrepareTransactionRequest, } from '../hooks/usePrepareTransactionRequest.js';
export { useProof, } from '../hooks/useProof.js';
export { usePublicClient, } from '../hooks/usePublicClient.js';
export { useReadContract, } from '../hooks/useReadContract.js';
export { useReadContracts, } from '../hooks/useReadContracts.js';
export { useReconnect, } from '../hooks/useReconnect.js';
export { useSendCalls, } from '../hooks/useSendCalls.js';
export { useSendCallsSync, } from '../hooks/useSendCallsSync.js';
export { useSendTransaction, } from '../hooks/useSendTransaction.js';
export { useSendTransactionSync, } from '../hooks/useSendTransactionSync.js';
export { useShowCallsStatus, } from '../hooks/useShowCallsStatus.js';
export { useSignMessage, } from '../hooks/useSignMessage.js';
export { useSignTypedData, } from '../hooks/useSignTypedData.js';
export { useSimulateContract, } from '../hooks/useSimulateContract.js';
export { useStorageAt, } from '../hooks/useStorageAt.js';
export { useSwitchChain, } from '../hooks/useSwitchChain.js';
export { 
/** @deprecated use `useSwitchConnection` instead */
useSwitchConnection as useSwitchAccount, useSwitchConnection, } from '../hooks/useSwitchConnection.js';
export { useTransaction, } from '../hooks/useTransaction.js';
export { useTransactionConfirmations, } from '../hooks/useTransactionConfirmations.js';
export { useTransactionCount, } from '../hooks/useTransactionCount.js';
export { useTransactionReceipt, } from '../hooks/useTransactionReceipt.js';
export { useVerifyMessage, } from '../hooks/useVerifyMessage.js';
export { useVerifyTypedData, } from '../hooks/useVerifyTypedData.js';
export { useWaitForCallsStatus, } from '../hooks/useWaitForCallsStatus.js';
export { useWaitForTransactionReceipt, } from '../hooks/useWaitForTransactionReceipt.js';
export { useWalletClient, } from '../hooks/useWalletClient.js';
export { useWatchAsset, } from '../hooks/useWatchAsset.js';
export { useWatchBlockNumber, } from '../hooks/useWatchBlockNumber.js';
export { useWatchBlocks, } from '../hooks/useWatchBlocks.js';
export { useWatchContractEvent, } from '../hooks/useWatchContractEvent.js';
export { useWatchPendingTransactions, } from '../hooks/useWatchPendingTransactions.js';
export { useWriteContract, } from '../hooks/useWriteContract.js';
////////////////////////////////////////////////////////////////////////////////
// Hydrate
////////////////////////////////////////////////////////////////////////////////
export { Hydrate, } from '../hydrate.js';
////////////////////////////////////////////////////////////////////////////////
// @wagmi/core
////////////////////////////////////////////////////////////////////////////////
export { ChainNotConfiguredError, ConnectorAccountNotFoundError, ConnectorAlreadyConnectedError, ConnectorChainMismatchError, ConnectorNotFoundError, ConnectorUnavailableReconnectingError, 
// Utilities
cookieStorage, cookieToInitialState, createConfig, createConnector, createStorage, 
// Transports
custom, deepEqual, deserialize, fallback, http, injected, mock, noopStorage, ProviderNotFoundError, parseCookie, SwitchChainNotSupportedError, serialize, unstable_connector, webSocket, } from '@wagmi/core';
////////////////////////////////////////////////////////////////////////////////
// Version
////////////////////////////////////////////////////////////////////////////////
export { version } from '../version.js';
